src/Symfony/Bridge/Doctrine/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.2
+---
+
+ * Add `NAME` constant to `UlidType` and `UuidType`
```

Comment:
maybe also create a docs PR that adjust https://symfony.com/doc/current/components/uid.html in the example (by using the constant instead of 'uuid'.

Reply:
Already done at symfony/symfony-docs#16866. :+1:

Reply:
Ah. I missed it in the description.